### DISCUSSION TITLE
Zoom in even if there are line break

### DISCUSSION THREAD

**Discussion by mafsi** [vslinko/obsidian-zoom#2] [Archive]
**Working:**

- item
  - item

---

**Not working:**

- item

  > this is a quote of an author
  > line 2 of the quote

    - this is a comment about the quote  `<--this line cannot be zoomed in`
      -  another comment to a comment `<--this line cannot be zoomed in`

**Comment by vslinko** [comment]
@mafsi yes, this happening because line breaks are unsupported yet. Please follow #38

**Reply by mafsi** [reply]
> 
> 
> @mafsi yes, this happening because line breaks are unsupported yet. Please follow #38

Thanks! Forget to tell you that I really appreciate this plugin!. Don't give up! Keep up the good work!

**Comment by vslinko** [comment]
Discussion was transferred from https://github.com/vslinko/obsidian-outliner to https://github.com/vslinko/obsidian-zoom

**Comment by vslinko** [comment]
@mafsi It should work with Obsidian Zoom plugin. Please check it out!

**Reply by mafsi** [reply]
There are 2 plugins now: obsidian outliner and obsidian zoom?

**Reply by vslinko** [reply]
@mafsi Yes, Outliner for lists manipulation and Zoom for lists and headings zooming.
